resources/docs/erp_abrufen_egk-source.adoc

```diff
@@ -0,0 +1,453 @@
+= E-Rezept API-Dokumentation für Apotheken (Einlösung mit Gesundheitskarte) image:gematik_logo.png[width=150, float="right"]
+include::./config-source.adoc[]
+
+Hier dokumentiert die gematik den Anwendungsfall der Einlösung eines E-Rezepts mittels Stecken der Gesundheitskarte
+
+toc::[]
+
+==  Anwendungsfall Liste einlösbarer E-Rezepte abrufen mit Stecken der Gesundheitskarte
+Mit diesem Anwendungsfall wird die Apotheke durch Übergabe und Stecken der eGK berechtigt, die Liste einlösbarer E-Rezepte vom E-Rezept-Fachdienst abzurufen. Zunächst muss ein VSD-Abruf über den Konnektor erfolgen, der mittels VSDM-Fachmodul eine Online-Gültigkeitsprüfung der eGK durchführt. In der Response gibt der Konnektor die von der eGK gelesenen Versichertenstammdaten VSD zurück.
+
+Aus den Persönlichen Versichertendaten (PD) des Versicherten extrahiert das AVS die KVNR, mit der nach einlösbaren E-Rezepten im E-Rezept-Fachdienst gesucht werden soll.
+
```

Comment:
Kann mit der einmal extrahierten KVNR nur während die Karte gesteckt ist am FD gesucht werden (bzw. innerhalb eines Zeitfensters) oder nach einmaliger Extraktion immer? Falls ja, soll das so sein?

Reply:
Der FD VSDM liefert die PZ zurück in der ein Zeittoken enthalten ist. Der FD prüft, ob der Zugriff in einem bestimmten Zeitfenster stattfindet. 
Nach Ablauf des Zeitfensters kann die Apotheke also nicht mehr zugreifen.